crates/ruff_linter/resources/test/fixtures/refurb/FURB181.py

```diff
@@ -0,0 +1,57 @@
+import hashlib
+from hashlib import (
+    blake2b,
+    blake2s,
+    md5,
+    sha1,
+    sha3_224,
+    sha3_256,
+    sha3_384,
+    sha3_512,
+    sha224,
+)
+from hashlib import sha256
+from hashlib import sha256 as hash_algo
+from hashlib import sha384, sha512, shake_128, shake_256
+
+# these will match
+
+blake2b().digest().hex()
+blake2s().digest().hex()
+md5().digest().hex()
+sha1().digest().hex()
+sha224().digest().hex()
+sha256().digest().hex()
+sha384().digest().hex()
+sha3_224().digest().hex()
+sha3_256().digest().hex()
+sha3_384().digest().hex()
+sha3_512().digest().hex()
+sha512().digest().hex()
+shake_128().digest(10).hex()
+shake_256().digest(10).hex()
+
+hashlib.sha256().digest().hex()
+
+sha256(b"text").digest().hex()
+
+hash_algo().digest().hex()
+
+# not yet supported
+h = sha256()
+h.digest().hex()
```

Comment:
It's not-impossible for us to detect this, you could look at what we do in `TRIO115` where we map from name to value. There's also a general utility in draft here (https://github.com/astral-sh/ruff/pull/8583), but not merged. Either way, not a requirement for merging, just making a mental note for myself if anything.

Reply:
Interesting, nice utility. Would be good to revisit once that PR is merged!

For myself, I took this easy rule to get back at ruff developing for a while. I'm keen to go for suggesting using the fluid interface (https://github.com/dosisod/refurb/pull/287). This impacts a lot of data science code I see.